src/components/BottomPanel/BottomPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
```

Comment:
just make it short `cn`
```suggestion
import cn from 'classnames';
```